### PR TITLE
fix profiles show more

### PR DESCRIPTION
**Pull request by octavioamu**
##### Checklist

- [ ] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [ ] Linter status: 100% pass
- [ ] Added relevant tests
- [ ] Tests pass and code coverage has not decreased
- [ ] Changes don't break existing behavior
- [ ] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [ ] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

##### Refers/Fixes

Fixes: #3889
###### Contributor

- [ ] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Tested all changes **locally**
- [ ] Verified existing functionality
- [ ] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging or local deployment

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

File: app/dashboard/templates/profiles/organization.html [hunk 1]
```diff
@@ -25,15 +25,17 @@
...
         <li><b>{{ sum_eth_on_repos|floatformat:2 }}</b> {% trans "ETH" %}</li>
         <li class="works_with_list">
-          {% if works_with_org|length != 0 %}
+          {% if works_with_org|length %}
             <span class="font-body"><i class="fa fa-code"></i> {% trans "Contributors" %}:</span>
-            {% for profile, num_times in works_with_org.items|slice:":5" %}
-              <a href="{% url 'profile' profile %} ">
-                <img src='{% avatar_url profile %}' title="<div class='tooltip-info tooltip-xs'>{{ profile }}: {{ num_times }} times</div>">
-              </a>
+            {% for profile, num_times in works_with_org.items %}
```
Comment (owocki) [inline app/dashboard/templates/profiles/organization.html]: but what if you work with lots of ppl, like gitcoin.co/profile/owocki does!!?

File: app/dashboard/templates/profiles/organization.html [hunk 2]
```diff
@@ -25,15 +25,17 @@
...
+                <a href="{% url 'profile' profile %} ">
+                  <img src='{% avatar_url profile %}' title="<div class='tooltip-info tooltip-xs'>{{ profile }}: {{ num_times }} times</div>">
+                </a>
+              {% endif %}
             {% endfor %}
-            {% if works_with_org|length > 5 %}
-              <div href="" class="plus-img" data-container="body" data-toggle="popover" data-html="true" data-placement="bottom" data-trigger="hover click" data-content='{% for profile in works_with_org.items|slice:"5:" %}<a href="{% url "profile" profile %}"><img src="{% avatar_url profile %}"/></a>{% endfor %}'><span>+10</span></div>
+            {% if works_with_org|length > 8 %}
+              <div href="" class="plus-img" data-container="body" data-toggle="popover" data-html="true" data-placement="bottom" data-trigger="hover click" data-content='{% for profile, num_times in works_with_org.items|slice:"5:" %}<a href="{% url "profile" profile %}"><img class="rounded-circle m-1" width="30" height="30" title="{{profile}}: {{ num_times }} times" src="{% avatar_url profile %}"/></a>{% endfor %}'><span>+{{works_with_org|length|add:"-8"}}</span></div>
```
Comment (thelostone-mc) [inline app/dashboard/templates/profiles/organization.html]: @octavioamu / @owocki  would it make sense to keep a fixed height and make it scrollable when number exceeds  8?

File: app/dashboard/templates/profiles/organization.html (hunk 1 above)
Comment (thelostone-mc) [reply app/dashboard/templates/profiles/organization.html]: @octavioamu / @owocki  Assuming the profile has worked with 15 folks 
would it make sense to 
- keep worked with card at a fixed height
- show 8 profiles you've worked with  `+ 7 More`
- On clicking  `+ 7 More` show all the profiles but make it scrollable

Comment (octavioamu) [reply app/dashboard/templates/profiles/organization.html]: That is right @thelostone-mc the popover show everyone

File: app/dashboard/templates/profiles/organization.html (hunk 2 above)
Comment (octavioamu) [reply app/dashboard/templates/profiles/organization.html]: I don't think will be a good experience to add a scroll in one line, is complicate to scroll and see it, I believe the popover is working fine for this objective

**Pull request merged**